docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -15,31 +15,50 @@ def make_predictions(las_client, event):
     document_id = event['documentId']
     model_id = os.environ['MODEL_ID']
     form_config_id = os.environ['FORM_CONFIG_ASSET_ID']
-    
+
     logging.info(f'Processing event:')
     logging.info(json.dumps(event, indent=2))
 
     form_config_asset = las_client.get_asset(form_config_id)
     form_config = json.loads(base64.b64decode(form_config_asset['content']))
-    
+
+    model_metadata = las_client.get_model(model_id).get('metadata', {})
+    logging.info(f'model metadata: {model_metadata}')
+
     output = {}
     needs_validation = True
 
     if not (predictions := event.get('predictions')):
-        try:
-            predictions = las_client.create_prediction(document_id, model_id).get('predictions')
-            logging.info(f'Created predictions {predictions}')
-        except Exception as e:
-            logging.exception(e)
-            predictions = []
-            output = {'predictions': predictions}
+        start_page = 0
+        predictions = []
+        while start_page is not None and start_page < model_metadata.get('maxPredictionPages', 100):
+            try:
+                current_prediction = las_client.create_prediction(
+                    document_id,
+                    model_id,
+                    preprocess_config={'startPage': start_page, 'maxPages': 3, 'imageQuality': 'HIGH'}
+                )
+                predictions.extend(current_prediction.get('predictions'))
+                start_page = current_prediction.get('nextPage', None)
```

Comment:
`None` is default from dict.get()
```suggestion
                start_page = current_prediction.get('nextPage')
```